pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -463,15 +485,15 @@ fi
 # or "boot"), or just build it and create a symlink "result" in the
 # current directory (for "build" and "test").
 if [ -z "$rollback" ]; then
-    echo "building the system configuration..." >&2
+    msg "building the system configuration..."
     if [[ "$action" = switch || "$action" = boot ]]; then
         if [[ -z $flake ]]; then
             pathToConfig="$(nixBuild '<nixpkgs/nixos>' --no-out-link -A system "${extraBuildFlags[@]}")"
         else
             pathToConfig="$(nixFlakeBuild "$flake#$flakeAttr.config.system.build.toplevel" "${extraBuildFlags[@]}" "${lockFlags[@]}")"
         fi
         copyToTarget "$pathToConfig"
-        targetHostCmd nix-env -p "$profile" --set "$pathToConfig"
+        targetHostCmd nix-env -p "$profile" --set "$pathToConfig" 2>&1 | msgPipe
```

Comment:
Do we not want to keep stdout and stderr separate for logging purposes? I think the journal treats them differently.

Reply:
Would making a msgErr and msgErrPipe be the best solution?

Reply:
That would work. But at this point maybe an `exec` which redirects stdout and stderr to separate `systemd-cat` commands would be a good way to capture *all* the output rather than only some commands?